frontend/dispatcher.lua

```diff
@@ -118,7 +118,7 @@ local settingsList = {
     clear_location_history = {category="none", event="ClearLocationStack", arg=true, title=_("Clear location history"), reader=true, separator=true},
     toc = {category="none", event="ShowToc", title=_("Table of contents"), reader=true},
     bookmarks = {category="none", event="ShowBookmark", title=_("Bookmarks"), reader=true},
-    bookmarks_search = {category="none", event="SearchBookmark", title=_("Bookmarks search"), reader=true},
+    bookmarks_search = {category="none", event="SearchBookmark", title=_("Bookmark search"), reader=true},
```

Comment:
You can also remove the plural in `bookmarks_search ` here and in `menu_items.bookmarks_search `